pkg/proxy/apis/config/scheme/testdata/KubeProxyConfiguration/roundtrip/default/v1alpha1.yaml

```diff
@@ -54,8 +54,8 @@ mode: ""
 nftables:
   masqueradeAll: false
   masqueradeBit: 14
-  minSyncPeriod: 1s
-  syncPeriod: 30s
+  minSyncPeriod: 0s
```

Comment:
@danwinship In conversion from internal -> v1alpha1 I'm currently setting `syncPeriod` and `minSyncPeriod` based on proxy-mode. Roundtrip will set these to zero values for other backends which I guess makes sense. 

Currently, we don't have a default value for `ipvs.minSyncPeriod` so the change to zero value is not visible (already zero) in this commit.

This would have been cleaner if 'omitempty' had been set for the v1alpha1 config.